pkg/controller/operators/olm/operator.go

```diff
@@ -396,7 +396,14 @@ func (a *Operator) handleClusterServiceVersionDeletion(obj interface{}) {
 
 	defer func(csv v1alpha1.ClusterServiceVersion) {
 		logger.Debug("removing csv from queue set")
-		a.csvQueueSet.Remove(csv.GetName(), csv.GetNamespace())
+		if err := a.csvQueueSet.Remove(csv.GetName(), csv.GetNamespace()); err != nil {
```

Comment:
Nice! It seems like it was missing from the original, but aren't CSV keys in other queues as well (e.g. `csvCopyQueue` and `csvGCQueue`)? Should we handle removing those as well? Could we wrap the delete handlers in our queue abstractions to have them automatically dropped after they return?

Reply:
These are good ideas for a follow up :) 

I was going to add it here but we don't even expose the other queues outside of their indexers, so that would be a bigger change